### ISSUE TITLE
Remove reference to "OneBusAway" from string

### ISSUE THREAD

**Issue by aaronbrethorst** [OneBusAway/onebusaway-ios#577]
It should be referred to as "the app" or maybe even pull the name from the Info.plist for white labeling purposes.
https://github.com/OneBusAway/onebusaway-ios/blob/1d8cba1ff728dda495765af8c3ad34602cff634c/OBAKit/Controls/FloatingPanel/OBAFloatingPanelController.swift#L56
